### ISSUE TITLE
Aria-label used on <div> in signup bar 

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4014]
## Area or Component
Sign up

## Quick Summary
The step indicator is a `<div>` but has an `aria-label` attached.

## Accessibility Violation
Elements must only use allowed ARIA attributes: https://dequeuniversity.com/rules/axe/4.4/aria-allowed-attr?application=AxeChrome

`aria-label` is not supported on `div` elements.

## Action Requested
Remove the `aria-label` from the div. If necessary, change the component so it's still understandable by screenreaders.

## Screenshots (optional)
The div is in here:
![Screen Shot 2022-07-20 at 3.27.32 PM.png](https://images.zenhubusercontent.com/604699209bfd9306195da0d7/64b8e73a-65cd-4142-929e-da5478ace96d)

## Source Code (optional)
[StepIndicator.tsx](https://github.com/CDCgov/prime-simplereport/blob/a520f60ba21ab1458b3093fbdfb96e61f40101f4/frontend/src/app/commonComponents/StepIndicator.tsx#L15-L15) ?

## Original Deque Issue
aria-allowed-attr | Ensures ARIA attributes are allowed for an element's role | The step indicator is a div but has the aria-label progress | Elements must only use allowed ARIA attributes | https://dequeuniversity.com/rules/axe/4.4/aria-allowed-attr?application=AxeChrome | serious | No | No |   | [".usa-step-indicator"] | Fix all of the following:  aria-label attribute is not well supported on a div with no valid role attribute. | <div class="usa-step-indicator usa-step-indicator--counters-sm usa-step-indicator--no-labels" aria-label="progress"> | cat.aria,wcag2a,wcag412 | rnv7@cdc.gov | Experian - Personal Details | https://stg.simplereport.gov/app/sign-up/identity-verification |   | 2022-07-18T17:31:34.919Z
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
